.github/workflows/build-deploy-pudl.yml

```diff
@@ -78,6 +78,10 @@ jobs:
       - name: Set up Cloud SDK
         uses: google-github-actions/setup-gcloud@v0
 
+      # Make sure the public and private zenodo-cache buckets are in sync
+      - name: Sync zenodo-cache buckets
+        run: gsutil -m rsync -d -r gs://zenodo-cache.catalyst.coop gs://zenodo-cache-internal
```

Comment:
Is there a reason you don't like using our namespace? The polluted global namespace gives me the willies.

Reply:
Meaning `catalyst.coop`?

Reply:
Yes, only we can create buckets that end with `.catalyst.coop` and so we have control over the namespace and it's easy to ensure that things are unique within it, whereas without the domain name suffix, it's a global namespace shared by all GCS buckets.

Reply:
Got it. How do you verify ownership of the new bucket?

Reply:
Yeah it's asking me to upload a [little hash file](https://www.google.com/webmasters/verification/verification?siteUrl=internal-zenodo-cache.catalyst.coop).

Reply:
Using UUIDs or other extremely unlikely to collide hashes in the global namespace seems common and reasonable (since they're designed to be globally unique) but for human readable individually named things it seems like working in our controlled subset of the namespace is more likely to allow us to keep things organized and avoid accidental collisions with other buckets that exist elsewhere due a typo or something.